search/tests/test_indexing.py

```diff
@@ -101,10 +110,15 @@ def test_sorting(self):
         )
         # expected position (res2, res3, res1)
         top_res = results[0]
+        sec_res = results[1]
         self.assertEqual(
             top_res.nr_views,
             res2.xa_nr_views
         )
+        self.assertEqual(
```

Comment:
The test of the sorting _would_ be implicit if you were comparing the IDs, but by only comparing `nr_views` they will always be the same, so you can't test the ordering.